### PR TITLE
Added my profile ManideepPaul

### PR DESCRIPTION
**Pull request by ManideepPaul**
Added my profile details.

### CODE REVIEW

File: Submissions/@ManideepPaul.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "links": {
+      "website": "[https://manideep-portfolio.vercel.app/]",
+      "linkedin": "https://www.linkedin.com/in/manideeppaul/",
+      "github": "https://github.com/ManideepPaul"
+    },
+    "jobTitle": "[Front-End Developer]",
+    "location": {
+      "city": "[Chhapra]",
+      "state": "[Bihar]",
+      "country": "[India]"
```
Comment (ProsperousHeart) [inline Submissions/@ManideepPaul.json:14]: Please remove the brackets - these are not necessary

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Please remove the brackets - these are not necessary

**Comment by ManideepPaul** [comment]
Creating new pull request

**Comment by ProsperousHeart** [comment]
FYI - no need to close and recreate in the future. :)

**Pull request closed without merging**